### ISSUE TITLE
Rosa command `rosa index foo.txt` fails

### ISSUE THREAD

**Issue by t-sin** [t-sin/rosa#21]
Why `nil` specified for `:external-format`?

```shell
$ rosa i hoge.txt
Unhandled TYPE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                  {1002F0E963}>:
  The value
    NIL
  is not of type
    (OR (CONS KEYWORD T) KEYWORD)
  when binding SB-IMPL::EXTERNAL-FORMAT

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1002F0E963}>
0: (OPEN #P"hoge.txt" :DIRECTION :INPUT :ELEMENT-TYPE BASE-CHAR :IF-EXISTS NIL :IF-DOES-NOT-EXIST NIL :EXTERNAL-FORMAT NIL :CLASS SB-SYS:FD-STREAM)
1: (ROSA/CLI::INDEX-LABELS #S(ROSA/CLI::OPTION :COMMAND :INDEX :FORMAT-TYPE NIL :PICK-LABEL NIL :PICK-ALL NIL :PICK-NTH NIL :PICK-TAIL NIL :PATHNAME #P"hoge.txt"))
2: (ROSWELL::ENTRY #<unused argument> "ROSA.CLI::MAIN")
3: (ROSWELL:RUN ((:ENTRY "ROSA.CLI::MAIN")))
4: ((FLET #:WITHOUT-INTERRUPTS-BODY-74 :IN SB-EXT:SAVE-LISP-AND-DIE))
5: ((LABELS SB-IMPL::RESTART-LISP :IN SB-EXT:SAVE-LISP-AND-DIE))

unhandled condition in --disable-debugger mode, quitting
```

**Comment by t-sin** [comment]
Encoding and EOL-style:

```shell
$ inq eol hoge.txt
LF
$ inq jp hoge.txt
UTF-8
```

**Comment by t-sin** [comment]
This caused by inquisitor change; UTF-8 keyword is renamed from `utf8` to `utf-8`.

**Comment by t-sin** [comment]
Fixed at 576593d033a872729e3e2f8730525d381e4e7840 .
